packages/libs/eda/src/lib/notebook/EdaNotebookAnalysis.tsx

```diff
@@ -60,9 +76,10 @@ export function EdaNotebookAnalysis(props: Props) {
       parentComputationId?: string
     ) {
       if (cell.type === 'compute') {
+        const appPlugin = plugins[cell.computationName];
         const computation = createComputation(
           cell.computationName,
-          {},
+          appPlugin.createDefaultConfiguration(),
```

Comment:
This is the line that fixes the app so that the user doesn't have to adjust the prevalence filter. The correlation app defines `createDefaultConfiguration` so that when it runs it sets the prefilter threshold(s). 

I should have made this a separate commit so it was easier to see!